Movies-MVP/Movies-MVP/Core/NetworkService/Protocols/NetworkServiceProtocol.swift

```diff
@@ -0,0 +1,22 @@
+// NetworkServiceProtocol.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+/// NetworkServiceProtocol
```

Comment:
```suggestion
/// NetworkServiceProtocol
```
Дока это не дублирование названий это описание для чего этот протокол/класс/структура